doc/source/remote_reading.rst

```diff
@@ -148,26 +148,3 @@ Resources
 See :class:`~satpy.readers.FSFile` for direct usage of `fsspec` with Satpy, and
 `fsspec documentation <https://filesystem-spec.readthedocs.io/en/latest/index.html>`_ for more details on connection options
 and detailes.
-
-
-Supported readers
```

Comment:
Do we have this info somewhere else?

Reply:
My understanding was this was being included as part of the work in #2108. CC @BENR0

Reply:
I should point out that this is being removed here because the documentation wouldn't render without warning with this included.

Reply:
@djhoese @mraspaud yes it it included. The reader table could also be included in the remote_reading document if desired.

Reply:
Ok thanks, I just merged that PR, so we should be fine now :)